### ISSUE TITLE
Unable to move between chart containers with accessibility module on IE

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#6461]
#### Expected behaviour
Should be able to TAB through chart and into next container.

#### Actual behaviour
Tab focus gets stuck on first container.

#### Live demo with steps to reproduce
http://jsfiddle.net/oysteinmoseng/sfx0j4Lp/

#### Affected browser(s)
IE11, possibly others (edge & firefox)

**Comment by DiiaKapoor** [comment]
Dear Øystein Moseng, 

Do you have any update on this issue? I have a very similar problem in Highchart 5.0.7 where tabbing gets stuck in the first chart container and specifically between two legend items. Also the legend items are not selected in the order of appearance. Can you please advise. I tested this in Chrome and Firefox. 
Thanks, Diia.

**Comment by sebastianbochan** [comment]
Hi @DiiaKapoor,
Could you simplify your demo on the jsfiddle? It will allow us to debug it and find a solution.
